QEfficient/generation/text_generation_inference.py

```diff
@@ -98,6 +98,9 @@ def latency_stats_bertstyle(
 
 
 def get_compilation_batch_size(qpc_path: str):
+    if qpc_path.endswith("/"):
+        logger.warning("Removing the slash('/')from given qpc path and looking for specializations.json in parent folder")
+        qpc_path = qpc_path.rstrip("/")
     qpc_base_path = os.path.dirname(qpc_path)
```

Comment:
Instead of this, can we just do: `qpc_base_path = os.path.dirname(os.path.normpath(qpc_path))` works for "/" and without. test it once pls

Reply:
yes, it works fine